drivers/storage/ide/uniata/scsi.h

```diff
@@ -1433,7 +1433,7 @@ typedef struct _READ_CAPACITY16_DATA {
     UCHAR Prot_EN:1;
     UCHAR RTO_EN:1;
     UCHAR Reserved:6;
-    UCHAR Reserved1[20];
+    UCHAR Reserved1[19];
```

Comment:
```suggestion
#ifndef __REACTOS__
    UCHAR Reserved1[20];
#else
    UCHAR Reserved1[19];
#end
```

Reply:
Are you sure this is good idea. Correct size of struct is 32 bytes.
https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf - page 158.

Reply:
The driver is third party, we use a ros-diff guard here (and already so in other places of this module).